src/blog/three-new-features-on-ganache-to-improve-your-developer-experience/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Three new Ganache features to improve your developer experience
+hide:
+  - navigation
+---
+
+**By [Kingsley Arinze](https://twitter.com/heydamali)**
+
+![The Merge and what it means for Truffle](./what-is-new-in-ganache.jpg)
+
+Since the release of Ganache v7.0 in January, we’ve mostly focused our attention on bug fixes and UX improvements. Recently, we decided to expand our focus to also include new features that will help improve the developer experience for our users.
+
+In addition, with the migration of Ethereum from PoW to PoS and the separation of the consensus layer from the execution layer, our users have been curious to know how Ganache is impacted and what actions they will need to take. 
+
+The simple answer to this very important question is: although [there are changes to how Ethereum clients will work post-merge](https://consensys.net/blog/ethereum-2-0/the-four-pillars-of-the-merge-to-proof-of-stake-how-ethereum-will-evolve), Ganache users do not need to do anything since these changes to Ethereum will not affect the user and dapp development experience.
+
+In this blog, we’ll touch on 3 of those new features that we've added to Ganache since the Ganache v7.0 release in January.
+
+## Zero-config mainnet forking now available in the browser
+
+Ganache v7.0 made it possible to run Ganache in the browser. Simply add this script to your HTML:
```

Comment:
If you "simply add this script" like this it won't work. Try out every example in this post and make sure they work, and the text descriptions make sense with the given examples. Please ping me again once you've made any necessary adjustments.